scripts/generate_bottle.sh

```diff
@@ -1,16 +1,16 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
-VERS=`sw_vers -productVersion | awk '/10\.14\..*/{print $0}'`
+VERS=`sw_vers -productVersion | awk '/10\.15\..*/{print $0}'`
 if [[ -z "$VERS" ]]; then
-   VERS=`sw_vers -productVersion | awk '/10\.15.*/{print $0}'`
+   VERS=`sw_vers -productVersion | awk '/11\.*.*/{print $0}'`
```

Comment:
The `\.*` in this regular expression does not make sense.
```bash
VERS=`sw_vers -productVersion | awk '/11\.*.*/{print $0}'`
```
Since the dot (`.`) is not required, it does nothing. We can either make the dot required...
```bash
VERS=`sw_vers -productVersion | awk '/11[.].*/{print $0}'`
```
-or-
```bash
VERS=`sw_vers -productVersion | awk '/11\..*/{print $0}'`
```
...or remove it altogether:
```bash
VERS=`sw_vers -productVersion | awk '/11.*/{print $0}'`
```

Reply:
I like your second way (VERS=`sw_vers -productVersion | awk '/11\..*/{print $0}'`)
 , I think the first dot after 11 is better to stay there, and then after this dot could be anything (.*) .